### ISSUE TITLE
[CI] Have a Meshery development environment on AWS

### ISSUE THREAD

**Issue by ramrodo** [meshery/meshery#2921]
#### Current Behavior
We don't have a dev environment of Meshery where we can access through the UI

#### Desired Behavior
Have Meshery UI deployed in AWS to be accessed for everyone

#### Implementation
Use EC2 instances or EKS service to deploy Meshery UI.

#### Reference:
- [Meshery Build & Release Strategy](https://docs.google.com/document/d/11nAxYtz2SUusCYZ0JeNRrOLIxkgmmbUVWz63MBZV2oE/edit#)

**Comment by shivaylamba** [comment]
Is this going to need a minikube installation to run in the dev enviornment?

**Comment by ramrodo** [comment]
> Is this going to need a minikube installation to run in the dev enviornment?

Not really. If we use EKS, the Kubernetes runner is already there. If not, we can use KinD.

**Comment by kumarabd** [comment]
@ramrodo Is this planned for anytime soon?

**Comment by ramrodo** [comment]
@kumarabd not soon but I hope before next month. Is it urgent?

**Comment by kumarabd** [comment]
> @kumarabd not soon but I hope before next month. Is it urgent?


nope it isn't a P1. Just checking in for an update.

**Comment by s1ntaxe770r** [comment]
Uhm, is it cool if i take this?, i could get KinD set up on EC2. Also must this be on AWS?

**Comment by ramrodo** [comment]
Hi @s1ntaxe770r , thanks for jumping on this. Sure, you can take this. It should be on AWS because we have an account set up there. Yes, you can do it with KinD + EC2, also I was considering using EKS that uses EC2 under the hood. But we can compare the two options and choose the better (or the cheaper).

**Comment by s1ntaxe770r** [comment]
I'm not sure what the budget is but, EKS would set us back at least $75 monthly , while ec2 + K3d  would be around $30, Also i think k3d would work better than kind in this case 
https://k3d.io/
